### DISCUSSION TITLE
Build from source on termux (aarch64)

### DISCUSSION THREAD

**Discussion by BufuWinner** [clangen/musikcube#507] [General]
I've been trying to build from source in termux (Linux) with the aarch64 architecture (not in the releases).
Many dependencies can't be installed from the package manager, by I tried my best to get everything. After many tries though I still get this error ->

`CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
LIBPANEL
    linked by target "musikcube" in directory /data/data/com.termux/files/home/musikcube/src/musikcube
-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.`

Does any one know which package sets the variable to something?
Sorry if it's a simple/stupid question but I can't go on with the installation...
Thanks in advance for any help!

**Comment by clangen** [comment]
`musikcube` has a number of dependencies that must be available on the system that are described here: https://github.com/clangen/musikcube/wiki/building -- unfortunately some of these may not be available in `termux`.

**Comment by ancientcatz** [comment]
It works fine
![Screenshot_2023-07-25-00-39-45-434_com termux](https://github.com/clangen/musikcube/assets/69200720/4ebcd11a-adb7-4bcd-abeb-8a97eee7ec76)
